src/test/regress/sql/seclabel.sql

```diff
@@ -0,0 +1,55 @@
+--
+-- SECLABEL
+--
+-- Test suite for SECURITY LABEL ON ROLE statements
+--
+
+-- first we remove one of the worker nodes to be able to test
+-- citus_add_node later
+SELECT citus_remove_node('localhost', :worker_2_port);
+
+CREATE ROLE user1;
+
+-- check an invalid label for our current dummy hook citus_test_object_relabel
+SECURITY LABEL ON ROLE user1 IS 'invalid_label';
+
+-- if we disable metadata_sync, the command will not be propagated
+SET citus.enable_metadata_sync TO off;
+SECURITY LABEL ON ROLE user1 IS 'citus_unclassified';
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('user1');
+
+RESET citus.enable_metadata_sync;
+
+-- check that we only support propagating for roles
+SET citus.shard_replication_factor to 1;
+CREATE TABLE a (a int);
+SELECT create_distributed_table('a', 'a');
+SECURITY LABEL ON TABLE a IS 'citus_classified';
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('a');
+DROP TABLE a;
+
+SET citus.log_remote_commands TO on;
+SET citus.grep_remote_commands = '%SECURITY LABEL%';
+
+-- then we run a security label statement which will use the same connection to the worker node
+-- it should finish successfully
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS 'citus_classified';
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS NULL;
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS 'citus_unclassified';
+
+RESET citus.log_remote_commands;
```

Comment:
Adding tests for other database object types would be useful. Additionally, local and distributed objects can be tested seperately to make sure that both local and distributed objects are working.

Reply:
I have added a test for a table object:
``` SQL
SECURITY LABEL ON TABLE a IS 'citus_classified';
```
And also for the case when metadata is not synced. Other local objects are already checked by the vanilla schedule.

Reply:
We may need to add database after merging my PR #7240 . 
So the only distributed objects are tables roles (databases after merge of #7240 ). What about views?

Reply:
Hmm, so you suggest adding a test for each distributed object that we could have. Okay, I will do that.

Reply:
> Hmm, so you suggest adding a test for each distributed object that we could have. Okay, I will do that.

This could be little bit extensive for this PR unless you've already added tests for them --not checked to `Files Changed` tab yet. :)

Because the check we have right now is already simple enough that it's ok to only test "roles" as well as one other type "that is not a role".

Reply:
Will there be seclabels on functions and procedures? If so how can we make sure that it is working for them. With this PR we can only say "Adds support of seclabels for tables and roles". Since we didn't test function procedure and other objects, we can not assure that they are working.

Reply:
> This could be little bit extensive for this PR

@onurctirtir True, I will not add tests for everything, but maybe some more tests with an object other than table.

> With this PR we can only say "Adds support of seclabels for tables and roles"

@gurkanindibay Technically, this PR only adds support for roles. For objects other than role (checked by an if statement if object_type is not role) then we simply print notice messages. So by testing something that is not a role we can assure (in a way) everything that is not a role is working.

Reply:
I have added tests for distributed functions and views as well. That should be enough.